marimbabot_ur5_flex_double_moveit_config/config/marimbabot_ur5.srdf

```diff
@@ -108,7 +108,7 @@
     <!--END EFFECTOR: Purpose: Represent information about an end effector.-->
     <end_effector name="s_model_tool0" parent_link="s_model_tool0" group="gripper" parent_group="arm"/>
     <end_effector name="mallet_head_1" parent_link="mallet_head_1" group="mallet_head_1_ee" parent_group="arm_mallets"/>
-    <end_effector name="mallet_head_2" parent_link="mallet_head_2" group="mallet_head_2_ee" parent_group="arm_mallets"/>
+    <!--<end_effector name="mallet_head_2" parent_link="mallet_head_2" group="mallet_head_2_ee" parent_group="arm_mallets"/>-->
```

Comment:
Why is this commented?

Reply:
Moveit throws an error if the robot has two end effectors. Reducing it to one does not change the planning code, but it reduces spam in the terminal. As we set the mallet position goals together with a bunch of other constraints via the ik options and not the normal API it doesn't change anything for us. It does change the behavior in rviz tho. You are no longer able to directly interact with both mallets in the 3d view.